mk/cargo.sh

```diff
@@ -170,6 +171,13 @@ case $target in
     export CARGO_TARGET_S390X_UNKNOWN_LINUX_GNU_LINKER=s390x-linux-gnu-gcc
     export CARGO_TARGET_S390X_UNKNOWN_LINUX_GNU_RUNNER="$qemu_s390x"
     ;;
+  sparc64-unknown-linux-gnu)
+    # Ultrasparc is the sparcv9 ISA, without this -mcpu compilers can
+    # default to lower ISA versions.
+    export CFLAGS_sparc64_unknown_linux_gnu="--sysroot=/usr/sparc64-linux-gnu -mcpu=ultrasparc"
```

Comment:
Well...I am a bit unsure about this `-mcpu-ultrasparc`. If this is something that everybody targeting sparc64 should do, then this should be done in cc-rs instead of here.

Reply:
The `-mcpu=ultrasparc` is definitely not necessary. sparc64 defaults to SPARCv9 which is what we want here.

See: https://gcc.gnu.org/onlinedocs/gcc/SPARC-Options.html